kubetest/gke.go

```diff
@@ -152,6 +154,8 @@ func newGKE(provider, project, zone, region, network, image, imageFamily, imageP
 			return nil, fmt.Errorf("--image-family and --image-project must be set for GKE deployment if --gcp-node-image=CUSTOM")
 		}
 	}
+
+	g.poolRe = regexp.MustCompile(fmt.Sprintf(poolTemplateRe, *gkeInstanceGroupPrefix))
```

Comment:
After moving regex compiling from package initialization to main code, please handle compile errors, as they would no longer be so obvious to catch.

Reply:
Done.